plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/refactoring/session/RefactoringSession.java

```diff
@@ -112,23 +127,109 @@ public Change getChange() {
         return change;
     }
 
-    public RefactoringStatus apply() {
+    /**@return instance of {@link org.eclipse.che.ide.ext.java.shared.dto.refactoring.RefactoringStatus}.
```

Comment:
Please, reformat it accordance to https://github.com/codenvy/codenvy-dev-resources/blob/master/che-codestyle/src/main/resources/che-codestyle-intellij.xml